t_malformed/test_malformed_headers.py

```diff
@@ -825,18 +801,6 @@ class EtagAlphabetBrangeTest(EtagAlphabetTest):
 block_action error reply;
 
 server ${server_ip}:8000;
-http_etag_brange 0x09 0x20-0x21 0x23-0x7e 0x80-0xff;
+http_etag_brange 0x21 0x23-0x7e 0x80-0xff;
```

Comment:
Please remove `EtagAlphabetBrangeTest` and `H2EtagAlphabetBrangeTest` tests. Now we have tests for `http_etag_brange` in other directory. (See `t_custom_branges`)